src/main/webapp/UserAuthScript.js

```diff
@@ -11,3 +11,18 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+/*
+* Purpose: recieves HTTP promise response from login.html, 
+* Once login redirects to comment.html
+*/
+var FETCH_LOGIN = '/login' 
+var LOGIN_CONTAINER = 'login-container';
+
+function login(){
```

Comment:
You're using lambda functions here to handle promises, then fully expanding out the functions in dialouge.js. For the sake of consistency, it might be better to choose either one and use it in both instances.